packages/components/src/mobile/bottom-sheet/navigation-container.native.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { BottomSheetContext, BottomSheetProvider } from '@wordpress/components';
+import { useState, useContext } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import { performLayoutAnimation } from '../layout-animation';
+
+function BottomSheetNavigationContainer( { children, animate } ) {
+	const [ height, setMaxHeight ] = useState( 1 );
+
+	const context = useContext( BottomSheetContext );
+	const setHeight = ( maxHeight ) => {
+		if ( height !== maxHeight && maxHeight > 50 ) {
+			if ( animate ) {
+				performLayoutAnimation( 300 );
```

Comment:
IIRC there is default animation duration equal 300 within `performLayoutAnimation` so there is no need to pass it.